file.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"fmt"
+	"net/url"
+	"path/filepath"
+	"strings"
+)
+
+type file struct {
+	Name string
+	URL  string
+}
+
+type files []file
+
+func (fs files) String() string {
```

Comment:
Fixed in:
https://github.com/bitrise-steplib/steps-generic-file-storage/pull/4/commits/54dbca202a1c8eeb6b35a416ec9780aaaf2a0a17

It's weird, because the golint was wired in already:
https://github.com/bitrise-steplib/steps-generic-file-storage/blob/master/bitrise.yml#L19

Reply:
another mistake this is the default String interface's implementation, probably this does not needs to be documented